src/schema/v2/homeView/HomeViewComponent.ts

```diff
@@ -2,13 +2,21 @@ import { OwnerType } from "@artsy/cohesion"
 import { GraphQLEnumType, GraphQLObjectType, GraphQLString } from "graphql"
 import { ResolverContext } from "types/graphql"
 
+/** Behaviors for this component. */
 export type HomeViewComponentBehaviors = {
+  /** Represents the behavior of the View All button */
   viewAll?: {
+    /** Route to navigate to, or null for default view-all behavior */
     href?: string | null
+
+    /** Text for the CTA of the View All button */
     buttonText?: string
+
+    /** TODO: confirm whether this correct or HomeViewSection.ownerType is correct */
     ownerType?: OwnerType
```

Comment:
@anandaroop Yeah, I can see how this is a bit confusing. `viewAll.href` + `viewAll.ownerType` are used together to navigate to an arbitrary screen _and_ give the client awareness of what _type_ of screen it represents for an analytics payload.

If `viewAll.href` is `null`, we can fallback to the client to implement default behavior, which could be to render a section-specific viewAll screen, in which case it seemed appropriate to specify `ownerType` on the top-level section configuration instead of requiring clients to drill down to `component > behaviors > viewAll > ownerType` in the query.